ml-agents/mlagents/trainers/stats.py

```diff
@@ -325,10 +332,15 @@ def get_stats_summaries(self, key: str) -> StatsSummary:
         if len(stat_values) == 0:
             return StatsSummary.empty()
 
+        if key == "Environment/Cumulative Reward":
```

Comment:
I'd say just always set `full = np.array(stat_values)`. This will play nicely with https://github.com/Unity-Technologies/ml-agents/pull/4788 (if I ever get time to finish it).